### PR TITLE
Make CI work

### PR DESCRIPTION
**Pull request by ekohl**
This includes https://github.com/Katello/katello-selinux/pull/36 since it's needed to compile.

### CODE REVIEW

**Comment by lzap** [comment]
It is red :-(

**Comment by ekohl** [comment]
Yes, I was also missing perl-podlators. Now it's green

File: katello.te [hunk 1]
```diff
@@ -35,6 +35,7 @@ require{
     type websockify_t;
     type candlepin_activemq_port_t;
     type tomcat_t;
+    type systemd_unit_file_t;
```
Comment (lzap) [inline katello.te:38]: Can you remove this commit, let's get it green, merge and I will rebase on top of that again.

Comment (lzap) [reply katello.te:38]: Or wait these are separate commits, we should be okay we can cherry pick it.

**Pull request merged**